docs/source/content/unsupervised/pca.rst

```diff
@@ -0,0 +1,117 @@
+represent them. For example, let's consider a small example dataset
+plotted below:
+
+.. figure:: _img/pca1.png
+
+   Ref: https://towardsdatascience.com/a-one-stop-shop-for-principal-component-analysis-5582fb7e0a9c
+
+Here we can see two direction pairs, represented by the red and green
+lines. In this scenario, the red line has a greater magnitude as the
+points are more clustered across a greater distance than with the
+green direction. Principal component analysis will use the vector
+with the greater magnitude to transform the data into a smaller
+feature space, reducing dimensionality. For example, the above graph
+would be transformed into the following:
+
+.. figure:: _img/pca2.png
+
+   Ref: https://towardsdatascience.com/a-one-stop-shop-for-principal-component-analysis-5582fb7e0a9c
+
+By transforming our data in this way, we've ignored a feature that
+is less important to our model - that is, higher variation along the
+green dimension will have a greater impact on our results than
+variation along the red.
+
+The mathematics behind principal component analysis are left out of
+this discussion for brevity, but if you're interested in learning
+about them we highly recommend visiting the references listed at the
+bottom of this page.
+
+Number of Components
+--------------------
+
+In the example above, we took a two-dimensional feature space and
+reduced it to a single dimension. In most scenarios though, you will
+be working with far more than two variables. Principal component
+analysis can be used to just remove a single feature, but it is often
+useful to reduce several. There are several strategies you can employ
+to decide how many feature reductions to perform:
+
+1. **Arbitrarily**
+
+   This simply involves picking a number of features to keep for your
+   given model. This method is highly dependent on your dataset and
+   what you want to convey. For instance, it may be beneficial to
+   represent your higher-order data on a 2D space for visualization.
+   In this case, you would perform feature reduction until you have
+   two features.
+
+2. **Percent of cumulative variability**
+
+   Part of the principal component analysis calculation involves
+   finding a proportion of variance which approaches 1 through each
+   round of PCA performed. This method of choosing the number of
+   feature reduction steps involves selecting a target variance
+   percentage. For instance, let's look at a graph of cumulative
+   variance at each level of PCA for a theoretical dataset:
+
+   .. figure:: _img/pca3.png
+
+      Ref: https://www.centerspace.net/clustering-analysis-part-i-principal-component-analysis-pca
+
+   The above image is called a scree plot, and is a representation
+   of the cumulative and current proportion of variance for each
+   principal component. If we wanted at least 80% cumulative variance,
+   we would use 6 principal components based on this scree plot.
```

Comment:
I would also add another example like "If we wanted 100% CV then we would use 12 principal components"

Reply:
for PCA you never really want 100% variance - if you end up with that it means all of your data is redundant and on the same line. ie, all rows are either the same or multiples of each other. i can make mention of that, though